governance/SIG Network Charter.md

```diff
@@ -1,97 +1,99 @@
 # SIG Network Charter
-
 This charter adheres to the Roles and Organization Management specified in <sig-governance>.
- Team information may be found in the <readme.md>
+Team information may be found in the [readme.md](https://github.com/o3de/sig-network/blob/main/README.md)
 
-## Overview of SIG
+Last Updated: **23rd Nov 2021**
 
-Two concise lines explaining what this SIG does with bullet points of the major responsibilities
+## Overview of SIG
+This sig is responsible for communication mechanisms in O3DE related to multiplayer networking and cloud networking. The SIG defines and owns
+communication protocols, methods and approaches to support O3DE client to server, as well as interaction points with cloud services.
 
-- Responsibility
+Responsibilities include:
+* Secure network transport including client+server interactions over TCP or UDP sockets.
+* Network state replication.
+* Restful HTTP(s) support.
 
 ## Goals
 
-- Major goals that SIG seeks to generally achieve
+- Major goals that SIG seeks to generally achieve: TBD
 
 ## Scope
 
 - Generalized overall scope of work
 
 ## In scope
 - Define standard for communication negotiation.
-- Define and maintain implementation to enable compression and secure communication between endpoints
-- Define and implement interface to network counters, monitoring, and statistics
-- Define and implement low level instrumentation APIs
+- Define and maintain implementation to enable compression and secure communication between endpoints.
+- Define and implement interface to network counters, monitoring, and statistics.
+- Define and implement low level instrumentation APIs.
+- Maintenance of the Multiplayer Sample project.
```

Comment:
Could we hyperlink "[Multiplayer Sample](https://github.com/o3de/o3de-multiplayersample)" -> https://github.com/o3de/o3de-multiplayersample?

Reply:
Done